openedxstats/apps/sites/tests.py

```diff
@@ -186,6 +282,14 @@ def test_add_language_that_already_exists(self):
             form.errors['name'], ['Language with this Name already exists.']
         )
 
+        # Will give an error message
+        response = self.client.post('/sites/add_language/', {'name': 'κόσμε'}, follow=True)
+        self.assertEqual(Language.objects.count(), 1)
+        storage = response.context['messages']
+        self.assertEqual(len(storage), 1)
+        for msg in storage:
+            self.assertIn("Oops! Something went wrong!", msg.message)
```

Comment:
Also, since you know storage only has one element, don't use a for-loop. Just access element `[0]`